app/src/main/java/com/kodeco/android/coordplot/MapComposable.kt

```diff
@@ -0,0 +1,74 @@
+package com.kodeco.android.coordplot
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Slider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
+
+
+@Composable
+fun MapComposable() {
+    var xSliderValue by remember { mutableStateOf(0.5f) }
+    var ySliderValue by remember { mutableStateOf(0.5f) }
+
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        MapGraph(xSliderValue, ySliderValue)
+        Slider(
+            value = xSliderValue,
+            onValueChange = { newValue ->
+                xSliderValue = newValue
+            },
+            valueRange = 0f..1.0f
+        )
+        Slider(
+            value = ySliderValue,
+            onValueChange = { newValue ->
+                ySliderValue = newValue
+            },
+            valueRange = 0f..1.0f
+        )
+        SliderValues(xSliderValue, ySliderValue)
+    }
+}
+@Composable
+fun MapGraph(xPercent: Float, yPercent: Float) {
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        Box(modifier = Modifier.size(300.dp)) {
```

Comment:
Don't forget to add a background color to the modifier here! When running the app it currently just shows a dot without the colored background for the surrounding box.

![Screenshot_20230905_205628](https://github.com/sgrem/CoordinatePlot/assets/2581931/92a43f6d-dab7-4db0-9bcc-2d7297d5ccd4)